subsys/bluetooth/controller/Kconfig

```diff
@@ -292,6 +293,19 @@ config BT_CTLR_DF_CONN_CTE_RSP
 config BT_CTLR_ADV_EXT_PDU_EXTRA_DATA_MEMORY
 	bool
 
+config BT_CTLR_SHORTER_CONNECTION_INTERVALS_SUPPORT
+	bool
+
+config BT_CTLR_SHORTER_CONNECTION_INTERVALS
+	bool "Shorter Connection Intervals"
```

Comment:
when this has a prompt and is also selected by `BT_LL_SOFTDEVICE` https://github.com/nrfconnect/sdk-nrf/blob/1fa91059a772100026bc0b72114dd89fe5f19e09/subsys/bluetooth/controller/Kconfig#L46 then there is a high risk this will suffer from the stuck symbol syndrome: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#stuck-symbols-in-menuconfig-and-guiconfig

Why is the symbol having a prompt ? / what are the use-cases where a user or sample needs to enable this setting ?

Reply:
Defining `BT_CTLR_SHORTER_CONNECTION_INTERVALS_SUPPORT` here gives us a head start until this is merged: https://github.com/zephyrproject-rtos/zephyr/pull/98901/files#diff-6d265bcf30897590772a25880fff70e895eee1836f95315d06146e08e90cb6baR136
Once this symbol is part of sdk-zephyr we'll drop the definition from here in the cherry-pick PR.

The support symbols are only selected by LLs that support a BLE feature, e.g. https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/bluetooth/controller/Kconfig.ll_sw_split#L8 does not so `BT_CTLR_SHORTER_CONNECTION_INTERVALS` can't be used with that link layer.